libs/langgraph/langgraph/utils/runnable.py

```diff
@@ -71,7 +71,7 @@ class StrEnum(str, enum.Enum):
         sys.intern("store"),
         (BaseStore, "BaseStore", inspect.Parameter.empty),
         CONFIG_KEY_STORE,
-        inspect.Parameter.empty,
+        None,
```

Comment:
@nfcampos i am a bit concerned about this one, but without this change you would get
```python
ValueError: Missing required config key '__pregel_store' for 'tools'.
```
when running `tool_node.invoke()` without passing store kwarg

Reply:
Right now I think raising an error is good actually?  

Would be better to have a specific user-facing one though. Maybe even handle better in the tool node? (like saying "please invoke within a graph where you've provided a store" or something of that nature